swift/Tests/PrivateKeyTests.swift

```diff
@@ -32,4 +32,15 @@ class PrivateKeyTests: XCTestCase {
 
         XCTAssertEqual(publicKey.data.hexString, "0499c6f51ad6f98c9c583f8e92bb7758ab2ca9a04110c0a1126ec43e5453d196c166b489a4b7c491e7688e6ebea3a71fc3a1a48d60f98d5ce84c93b65e423fde91")
     }
+
+    func testPrivateKeyToWIF() {
+        // taken from https://en.bitcoin.it/wiki/Wallet_import_format
+        let wif = "5HueCGU8rMjxEXxiPuD5BDku4MkFqeZyd4dZ1jvhTVqvbTLvyTJ"
+        let data = Data(hexString: "0x0C28FCA386C7A227600B2FE50B7CAE11EC86D3BF1FBE471BE89827E19D72AA1D")!
+        let key = PrivateKey(data: data)!
```

Comment:
are these tests covered in C++ too?

Reply:
I was considering remove `wif` things, not that useful, @alejandro-isaza do you agree?

Reply:
Yes, I don't think we use it.